pkg/kubernetes/revealable.css

```diff
@@ -0,0 +1,46 @@
+.revealable {
+    white-space: nowrap !important;
+    text-overflow: ellipsis;
+    overflow: hidden !important;
+}
+
+.revealable.clickable {
+    cursor: pointer;
+}
+
+.revealed {
+    white-space: normal !important;
+    overflow: visible !important;
+    cursor: pointer;
+    overflow-wrap: break-word;
+}
+
+.masked {
+    color: transparent;
+    position: relative;
+    text-ellipsis: none !important;
+    cursor: pointer;
+    max-width: 1;
+    display: inline-block;
+}
+
+.masked:after {
+    content: "●●●●";
+    position: absolute;
+    left: 0px;
+    font-weight: bold;
+    letter-spacing: 2px;
+    padding-left: 2px;
+    color: black;
+    cursor: pointer;
+    line-height: 1.5;
+    opacity: 0.5;
```

Comment:
Yes, that's intentional. The dots are too big, and stand out too much if they are of identical color.